stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -708,13 +712,15 @@ public class MessageInputView : ConstraintLayout {
 
     private fun refreshControlsState() {
         with(binding) {
-            val commandMode = messageInputFieldView.mode is MessageInputFieldView.Mode.CommandMode
+            val isCommandMode = messageInputFieldView.mode is MessageInputFieldView.Mode.CommandMode
+            val isEditMode = messageInputFieldView.mode is MessageInputFieldView.Mode.EditMessageMode
             val hasContent = messageInputFieldView.hasValidContent()
             val hasValidContent = hasContent && !isMessageTooLong()
 
-            attachmentsButton.isVisible = messageInputViewStyle.attachButtonEnabled && !commandMode
-            commandsButton.isVisible = shouldShowCommandsButton() && !commandMode
-            commandsButton.isEnabled = !hasContent
+            attachmentsButton.isVisible = messageInputViewStyle.attachButtonEnabled && !isCommandMode
+            attachmentsButton.isVisible = messageInputViewStyle.attachButtonEnabled && !isEditMode
```

Comment:
Can we combine these two conditions, so that we don't change the same button twice?

It might be that one of these needs to be `isEnabled` instead of `isVisible`

Reply:
I think it would be better to make it gone instead of disabled... Sometimes the disabled color it not very clear and it confuses the user.